### PR TITLE
[8.0][WIP] Add float head in tree view

### PR DESCRIPTION
**Pull request by angelmoya**
I'm trying to set float head in tree view.

I'm ussing this javascript lib: http://mkoryak.github.io/floatThead/

I just created an Issue, #246

### CODE REVIEW

File: web_tree_float_head/static/src/js/web_table_head_float.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+openerp.web_table_head_float = function(instance)
+{
+    var $table = $("table.oe_list_content");
+	$table.floatThead({
+	    scrollContainer: function($table){
+	        return $table.closest('.wrapper');
+	    }
+	});
+}
```
Comment (angelmoya) [inline web_tree_float_head/static/src/js/web_table_head_float.js]: This code works in javascript console, but obviously is not correctly called

**Comment by legalsylvain** [comment]
Hi @amoya-dx,

Intererested feature indeed. I have no idea why the function is not correctly called. I'll take a look during this week.

kind regards.

**Comment by legalsylvain** [comment]
BTW, did you take a look on #133 ?

**Comment by angelmoya** [comment]
I was looking for this functionality but did not seen this PR before. I will take a look this PR

File: web_tree_float_head/README.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Display float head in tree view
+=====================================
+
+This module set head as float in tree view.
```
Comment (elicoidal) [inline web_tree_float_head/README.rst]: The description is cryptic.
Can you elaborate so that we can improve it?

Comment (angelmoya) [inline web_tree_float_head/README.rst]: Ok, work is in progess.

Best: This module sets the header always visible in the tree view .

File: web_tree_float_head/static/src/js/web_table_head_float.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+//
+//############################################################################
+
+openerp.web.ListView.include({
+    /**
+     * Override all list view to add sticky header behavior
+     **/
+    init: function(parent, dataset, view_id, options) {
+        var self = this;
+        self._super(parent, dataset, view_id, options);
```
Comment (lmignon) [inline web_tree_float_head/static/src/js/web_table_head_float.js:27]: you should modify your code to only call the plugin once the widget is rendered. Can you try with this code?

``` javascript
init: function(parent, dataset, view_id, options) {
     var self = this;
     self._super(parent, dataset, view_id, options);
     self.on("list_view_loaded", self, function() {
        var $table = self.$el.find("table.oe_list_content");
        $table.floatThead({
            scrollContainer: function($table){
                return $table.closest('.wrapper');
            }
        });
     },
```

no tested...

**Comment by fossoult** [comment]
I tried in V8 but it doesn't work event after fix the js bug. Is it working for you ?

**Comment by pedrobaeza** [comment]
Any update on this?

**Comment by pedrobaeza** [comment]
@angelmoya, are you going to work on this anytime or should I close it?

**Pull request closed without merging**